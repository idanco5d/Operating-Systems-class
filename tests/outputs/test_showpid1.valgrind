==23093== Memcheck, a memory error detector
==23093== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==23093== Using Valgrind-3.20.0 and LibVEX; rerun with -h for copyright info
==23093== Command: /mnt/c/Users/illus/os/hw/1/smash
==23093== Parent PID: 23088
==23093== 
==23093== 
==23093== FILE DESCRIPTORS: 4 open (3 std) at exit.
==23093== Open file descriptor 3: /mnt/c/Users/illus/os/hw/1/tests/outputs/test_showpid1.valgrind
==23093==    <inherited from parent>
==23093== 
==23093== 
==23093== HEAP SUMMARY:
==23093==     in use at exit: 0 bytes in 0 blocks
==23093==   total heap usage: 16 allocs, 16 frees, 78,968 bytes allocated
==23093== 
==23093== All heap blocks were freed -- no leaks are possible
==23093== 
==23093== For lists of detected and suppressed errors, rerun with: -s
==23093== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
