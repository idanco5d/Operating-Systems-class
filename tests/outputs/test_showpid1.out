smash> smash pid is 9699
smash> 